### PR TITLE
Add initial 200 level labs

### PR DESCRIPTION
**Pull request by flands**
- Fix copy commands
- Add more links
- Add more debug information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,20 +32,20 @@ $ docker-compose up --build
...
+```bash
+curl -i localhost:3000/hello
+curl -i localhost:8080/hello/proxy/python/node/java
 ```
 
 Other included components:
 
 - Once instrumented with OpenTelemetry, view traces in Jaeger:
-http://localhost:16686.
+`http://localhost:16686`.
```
Comment (jkwatson) [inline README.md]: Won't this make the link non-clickable in GitHub?

File: docker/lab200.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+docker-compose YAML file. In addition, an [initial configuration
+file](configs/config.yaml) exists. In this lab, you will experience sending
+different synthetic data to the collector and modifying the collector's
+configuration to change behavior.
+
+For all parts of Lab 200, the Collector will be started with:
+
+```bash
+cd docker
+docker-compose run -p13133:13133 -p55679:55679 -p8081:8081 \
```
Comment (jkwatson) [inline docker/lab200.md:13]: Any reason not to expose the ports in the docket-compose yaml and use the `up` command with just the single service?

File: docker/lab200.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+                   -p8888:8888   -p9091:9090   -p9411:9411 otelcol
+```
+
+Once running, you can:
+
+- See the collector's own metrics at `http://localhost:8888/metrics`
```
Comment (jkwatson) [inline docker/lab200.md]: It would be nice to keep these clickable when viewing in GH directly.

File: README.md (hunk 1 above)
Comment (flands) [reply README.md]: Why yes!

File: docker/lab200.md (hunk 2 above)
Comment (flands) [reply docker/lab200.md:13]: So I tried that but it did not expose the ports...

**Pull request merged**